driver/razermouse_driver.c

```diff
@@ -1559,6 +1559,154 @@ static ssize_t razer_attr_read_mouse_dpi(struct device *dev, struct device_attri
     return sprintf(buf, "%u:%u\n", dpi_x, dpi_y);
 }
 
+/**
+ * Write device file "scroll_mode"
+ *
+ * Set scroll wheel mode on the device
+ *
+ * Status Trans Packet Proto DataSize Class CMD Args
+ * 00     1f    0000   00    02       02    14  0100    | SET SCROLL WHEEL ACCELERATION (VARSTR, TACTILE)
+ * 00     1f    0000   00    02       02    14  0101    | SET SCROLL WHEEL ACCELERATION (VARSTR, FREESPIN)
+ */
+static ssize_t razer_attr_write_scroll_mode(struct device *dev, struct device_attribute *attr, const char *buf, size_t count)
+{
+    struct razer_mouse_device *device = dev_get_drvdata(dev);
+    struct razer_report report = get_razer_report(0x02, 0x14, 0x02);
```

Comment:
please make new methods in razerchromacommon.c / razercommon.c to separate this. Applies to all the methods you added

Reply:
Not sure how I'd missed that rather obvious layering there, but alas...

Should be better now. I've put the methods in `razerchromacommon.c` because `razercommon.c` seemed to have low-level things that the scroll stuff just didn't belong to.